core/src/main/scala/com/databricks/labs/remorph/generators/sql/DataTypeGenerator.scala

```diff
@@ -23,7 +23,7 @@ object DataTypeGenerator {
     case ir.DecimalType(precision, scale) =>
       val arguments = precision.toSeq ++ scale.toSeq
       if (arguments.isEmpty) {
-        "DECIMAL"
+        "DECIMAL(38, 0)"
```

Comment:
```suggestion
        "DECIMAL(10, 0)"
```

> p: Optional maximum precision (total number of digits) of the number between 1 and 38. The default is 10. s: Optional scale of the number between 0 and p. The number of digits to the right of the decimal point. The default is 0.



https://docs.databricks.com/en/sql/language-manual/data-types/decimal-type.html

Reply:
Reverted, and changed in the Parser Side to accommodate Snow behavior to keep the generation consistent with our python parser.

https://docs.snowflake.com/en/sql-reference/data-types-numeric